xcodeproj/internal/xcodeproj_rule.bzl

```diff
@@ -744,8 +744,8 @@ targets.
             compile_params.append(xcode_target._c_params)
         if xcode_target._cxx_params:
```

Comment:
Can you do the same for the c/cxx params as well, since they are referenced here?

Reply:
Oh, geeze, whoops. Good point.